x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/saved_object/engine_descriptor_type.ts

```diff
@@ -31,6 +31,6 @@ export const entityEngineDescriptorType: SavedObjectsType = {
   name: entityEngineDescriptorTypeName,
   indexPattern: SECURITY_SOLUTION_SAVED_OBJECT_INDEX,
   hidden: false,
-  namespaceType: 'multiple-isolated',
```

Comment:
I am not sure what the right move is here, we have the namespace in the ID right? And I cant imagine an engine configuration being shared by mutliple spaces. So maybe single is more like what we want? But maybe multiple does no harm and leaves us with the option to share it in the future. Im unsure.

Reply:
reverted to multiple-isolated as discussed at standup